.travis.yml

```
@@ -6,6 +6,7 @@ php:
     - 7.2
     - 7.3
     - 7.4
+    - 8.0
```

```suggestion
    - '8.0snapshot'
```

Travis does not have stable 8.0 builds yet.